src/otx/config/model/mmpretrain.yaml

```diff
@@ -0,0 +1,26 @@
+_target_: otx.core.model.module.classification.OTXClassificationLitModule
+
+optimizer:
+  _target_: torch.optim.Adam
+  _partial_: true
+  lr: 1e-3
+  weight_decay: 0.0
+
+scheduler:
+  _target_: torch.optim.lr_scheduler.ReduceLROnPlateau
+  _partial_: true
+  mode: min
+  factor: 0.1
+  patience: 10
+
+# scheduler:
+#   _target_: torch.optim.lr_scheduler.CosineAnnealingLR
+#   _partial_: true
+#   T_max: ${trainer.max_epochs}
```

Comment:
Can we remove these lines?

Reply:
Sure, no matter

Reply:
Done. https://github.com/openvinotoolkit/training_extensions/pull/2678/commits/80ebb392bf2ca7e67f4b1806c9549de3062e3eed